config/local.json.example

```diff
@@ -27,6 +27,13 @@
     "apiKey": "",
     "secretKey": ""
   },
+  "email": {
+    "fromEmail": "info@chronicle.firefox.com",
```

Comment:
Email sent with this from address never arrived in my gmail account. I don't know what a good solution is here, though, so we can just leave it.